internal/cephfs/core/volume.go

```diff
@@ -32,12 +34,17 @@ import (
 	"github.com/ceph/go-ceph/rados"
 )
 
-// clusterAdditionalInfo contains information regarding if resize is
-// supported in the particular cluster and subvolumegroup is
-// created or not.
-// Subvolumegroup creation and volume resize decisions are
-// taken through this additional cluster information.
-var clusterAdditionalInfo = make(map[string]*localClusterState)
+var (
+	// clusterAdditionalInfo contains information regarding if resize is
+	// supported in the particular cluster and subvolumegroup is
+	// created or not.
+	// Subvolumegroup creation and volume resize decisions are
+	// taken through this additional cluster information.
+	clusterAdditionalInfo = make(map[string]*localClusterState)
+	// clusterAdditionalInfoMutex is used to synchronize access to
+	// clusterAdditionalInfo map.
+	clusterAdditionalInfoMutex = sync.Mutex{}
```

Comment:
Would a `sync.RWMutex` not be better?

If you only mean to protect against concurrent writes, please state it clearly.